### PR TITLE
Basic 'User Profile page' created

### PR DESCRIPTION
**Pull request by ishwaryaPaderborn**
@bhermann  and @nguyenLisa, here is a very simple and basic 'User profile' page implementation. Kindly review them and based on your suggestions and feedback I would improvise the page accordingly. And parallelly I'm also trying to customize the page and thinking about adding more fields that suffice our requirements.

### CODE REVIEW

**Comment by bhermann** [comment]
Hello @ishwaryaPaderborn . I had a look at the page. 
I think we will not need a field for the organization, as only administrators will use this.
Currently, we do not have a planned notification feature, so the radio buttons are not necessary as well. With four options, a dropdown would also be better suited. 
What we might need earlier (please discuss in the group) are fields for password change.

File: client/src/app/dashboard/user-profile/user-profile.component.ts [hunk 1]
```diff
@@ -7,6 +8,23 @@ import { Component, OnInit } from '@angular/core';
 })
 export class UserProfileComponent implements OnInit {
 
+	notification_level: string;
```
Comment (janniclas) [inline client/src/app/dashboard/user-profile/user-profile.component.ts]: please stick to the naming convention of camel case :)

**Comment by ishwaryaPaderborn** [comment]
@bhermann Thank you for the review. As suggested I will remove the 'Organization' field permenently and also the 'Notification level' for now. Also, I will discuss the 'Change Password' functionality with the team tommorrow and update the UI accordingly.

**Comment by ishwaryaPaderborn** [comment]
Closing this for now. Will re-open when more functionalities for 'user profile' is implemented.

**Pull request closed without merging**